packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Changed
+- SF-1271: The `composeRequest` function now uses the `attachSessionId` function to add sessionIds to every StoreFront request.
```

Comment:
Include trailing `()` to signify that the snippet is a function. Drop 'function' word.

Reply:
Will change.